3_extracting_features/README.md

```diff
@@ -0,0 +1,104 @@
+# 2. Extract NF1 Features
+
+In this module, I present the pipeline for extracting features from the NF1 data.
+
+### Feature Extraction
+
+To extract features from the NF1 data, I use [DeepProfiler](https://github.com/cytomining/DeepProfiler), commit [`8752f69`](https://github.com/cytomining/DeepProfiler/commit/8752f69686a0b0c53d4e829d598409506bac59f4).
+
+Based off of previous projects in the lab, I decided to use a [pretrained model](https://github.com/broadinstitute/luad-cell-painting/tree/main/outputs/efn_pretrained/checkpoint) from the [LUAD Cell Painting repository](https://github.com/broadinstitute/luad-cell-painting) with DeepProfiler.
+DeepProfiler has the function to be able to train and create your own model (also named `checkpoint`) which I would like to test in the future.
+
+The config files were made based off the one used in the [same repository from above](https://github.com/broadinstitute/luad-cell-painting).
+The following changes were made to each config file, which are [NF1_nuc_config.json](DP_files/NF1_nuc_config.json) and [NF1_cyto_config.json](DP_files/NF1_cyto_config.json).
+
+**Both:**
+- `"Allele" -> "Genotype` In the LUAD study, alleles were compared across cell painting images. For the NF1 data, the genotypes of the NF1 gene are compared.
+- `dataset: images: {file format: tif, bits: 16, width: 1080, height: 1080} -> dataset: images: {file format: tiff, bits: 8, width: 1224, height: 904}`: The image details are changed to reflect the NF1 data.
+- `prepare: implement: true -> prepare: implement: false` We do not prepare the NF1 data with illumination correction (already done) or compression with Deep Profiler.
```

Comment:
oh interesting, DeepProfiler can perform IC?

Reply:
There is an ability to do that in the training portion which Roshan and I didn't use in Cell Health and I continued it here.

Reply:
ah, I see! I wasn't aware of this functionality - do you know what IC approach it uses?

Reply:
No idea currently. Definitely something we can look into in the future!